tests/operators/test_autodiff.py

```diff
@@ -94,9 +96,9 @@ def test_adjoint_and_grad(operator, interface):
             )
             adj_data_ndft = torch.sum(smaps.conj() * adj_data_ndft_smaps, dim=0)
         else:
-            adj_data_ndft = (
-                ndft_matrix(operator).conj().T @ ksp_data.flatten()
-            ).reshape(img_data.shape)
+            adj_data_ndft = torch.matmul(
+                ndft_matrix(operator).conj().T, ksp_data.T
+            ).T.reshape(img_data.shape)
```

Comment:
To be honest, I am a little bit confused why @ has been replaced by the torch.matmul... Does torch.matmul cost less time or memory?